source/retryable-writes/tests/README.rst

```diff
@@ -360,24 +360,27 @@ and sharded clusters.
 
 #. Test that drivers properly retry after encountering PoolClearedErrors. This
    test MUST be implemented by any driver that implements the CMAP
-   specification. This test requires MongoDB 4.2.9+ for ``blockConnection`` support in the failpoint.
+   specification.
+
+   This test requires MongoDB 4.3.4+ for both the ``errorLabels`` and
```

Comment:
Since no one should be testing with any 4.3 release, can we just bump to 4.4?

Reply:
> Why 4.3.4 here rather than 4.3.1

4.3.4 is the first release where both `errorLabels` _and `blockConnection` were available (see: [SERVER-41070](https://jira.mongodb.org/browse/SERVER-41070)).

> Since no one should be testing with any 4.3 release, can we just bump to 4.4?

IMO, using exact versions when they're accessible is helpful to preserve history. Seeing the versions before I started on this PR is what clued me in that https://github.com/mongodb/mongo/wiki/The-%22failCommand%22-fail-point and the Unified Test Format spec needed to be updated.

I suppose something to consider down the line would be to periodically bump server versions in spec/prose tests after final versions are released (e.g. after 7.0 is cut, we should have no server version requirements for pre-7.0 unstable releases in existing tests) -- but I don't need to be involved in that.